### ISSUE TITLE
Hello, thank you for your excellent contributions. I have a question. I want to ask, I always get the error of object has no attribute '_up_kwargs' when I add the strippooling module to my own network. What is the reason? My code is as follows, 

### ISSUE THREAD

**Issue by houqian2180320171** [Andrew-Qibin/SPNet#3]
def __init __ (self, nDenselayer, channels, growth, norm_layer = nn.BatchNorm2d, ** kwargs):
         super (RDB1, self) .__ init __ ()
         modules = []
         channels_buffer = channels
         for i in range (nDenselayer):
             modules.append (make_dense1 (channels_buffer, growth, 3, norm_layer, self._up_kwargs))

**Comment by swjtulinxi** [comment]
hi，i can not find the code of spm in you model,can you show me where is it? thanks

**Comment by houqb** [comment]
You can find it in models/resnet.py

**Comment by houqb** [comment]
> def **init __ (self, nDenselayer, channels, growth, norm_layer = nn.BatchNorm2d, ** kwargs):          super (RDB1, self) .** init __ ()
>          modules = []
>          channels_buffer = channels
>          for i in range (nDenselayer):
>              modules.append (make_dense1 (channels_buffer, growth, 3, norm_layer, self._up_kwargs))

you can ignore it.
